com.unity.render-pipelines.universal/ShaderLibrary/MetaInput.hlsl

```diff
@@ -1,70 +1,85 @@
 #ifndef UNIVERSAL_META_PASS_INCLUDED
 #define UNIVERSAL_META_PASS_INCLUDED
 
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/MetaPass.hlsl"
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl"
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl"
 
-CBUFFER_START(UnityMetaPass)
-// x = use uv1 as raster position
-// y = use uv2 as raster position
-bool4 unity_MetaVertexControl;
-
-// x = return albedo
-// y = return normal
-bool4 unity_MetaFragmentControl;
-CBUFFER_END
+struct Attributes
+{
+    float4 positionOS   : POSITION;
+    float3 normalOS     : NORMAL;
+    float2 uv0          : TEXCOORD0;
+    float2 uv1          : TEXCOORD1;
+    float2 uv2          : TEXCOORD2;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
 
-float unity_OneOverOutputBoost;
-float unity_MaxOutputValue;
-float unity_UseLinearSpace;
+struct MetaVaryings
+{
+    float4 positionCS   : SV_POSITION;
+    float2 uv           : TEXCOORD0;
+#ifdef EDITOR_VISUALIZATION
+    float2 VizUV        : TEXCOORD1;
+    float4 LightCoord   : TEXCOORD2;
+#endif
+};
 
-struct MetaInput
+struct Varyings
```

Comment:
I'm not sure why the rename to MetaVaryings is needed and I worry that now user could be using this as thsi is part of `ShaderLibrary` and will have compiler errors. At the bare least you could forward declare the old to the new MetaVaryings if needed.